debug/wrappers.h

```diff
@@ -34,8 +34,9 @@
 #define WRAPPER_DYNHOOK             2
 #define WRAPPER_ARM_INJECTION       3
 #define WRAPPER_THUMB_INJECTION     4
+#define WRAPPER_DALVIK              5
```

Comment:
Those `WRAPPER_*` defines could be turned into an `enum` and used below as type (instead of `int wrapper_type`).